database-plugins/src/main/java/io/cdap/plugin/DBUtils.java

```diff
@@ -196,7 +211,8 @@ public static DriverCleanup ensureJDBCDriverIsAvailable(Class<? extends Driver>
   }
 
   // given a sql type return schema type
-  private static Schema getSchema(String typeName, int sqlType, int precision, int scale) throws SQLException {
+  private static Schema getSchema(String typeName, int sqlType, int precision, int scale, String columnName,
```

Comment:
This part is different between release 2.3 and 2.4. Develop has columnName and release 2.3 does not. Is there a PR cherrypick missing that needs to be here?